src/response_sender.rb

```diff
@@ -0,0 +1,31 @@
+class ResponseSender
+  attr_reader :response, :message, :bot
+
+  def initialize(handler, *args)
+    @bot = handler.bot
+    @message = handler.message
```

Comment:
I don't see you're using `@message` variable otherwise as getting chat id.
Maybe it's better to replace with `@chat_id`?